ComputeDataDrivenCrossSection.py

```diff
@@ -23,17 +19,48 @@
 parser.add_argument('--energy', metavar='text', default='5.02', help='energy (5.02)')
 parser.add_argument('--centrality', metavar='text', default='010', help='centrality (010, 3050)')
-parser.add_argument("--prompt", action='store_true', help='flag to compute prompt cross section', default=False)
-parser.add_argument("--FD", action='store_true', help='flag to compute FD cross section', default=False)
-parser.add_argument("--Dplus", action='store_true', help='flag to compute D+ cross section', default=False)
-parser.add_argument("--Ds", action='store_true', help='flag to compute Ds cross section', default=False)
+
+groupComponent = parser.add_mutually_exclusive_group(required=True)
+groupComponent.add_argument("--prompt", action='store_true', help='flag to compute prompt cross section', default=False)
+groupComponent.add_argument("--FD", action='store_true', help='flag to compute FD cross section', default=False)
+
+groupParticle = parser.add_mutually_exclusive_group(required=True)
+groupParticle.add_argument("--Dplus", action='store_true', help='flag to compute D+ cross section', default=False)
+groupParticle.add_argument("--Ds", action='store_true', help='flag to compute Ds cross section', default=False)
+groupParticle.add_argument("--Lc", action='store_true', help='flag to compute Lc cross section', default=False)
+
 parser.add_argument("--batch", action='store_true', help='suppress video output', default=False)
+
+groupCorrelation = parser.add_mutually_exclusive_group()
+groupCorrelation.add_argument("--corr", action='store_true',
+    help='assume raw yield and (prompt/feed-down) fraction uncertanties as correlated', default=True)
+groupCorrelation.add_argument("--uncorr", action='store_true',
+    help='assume raw yield and (prompt/feed-down) fraction uncertanties as uncorrelated', default=False)
+groupCorrelation.add_argument("--anticorr", action='store_true',
+    help='assume raw yield and (prompt/feed-down) fraction uncertanties as anti correlated', default=False)
 args = parser.parse_args()
 
 # Define arguments (systematic uncertainties from AliHFSystErr)
+from ROOT import TFile, TCanvas, TLegend, TGraphErrors, gROOT  # pylint: disable=import-error,no-name-in-module
+from ROOT import AliHFSystErr  # pylint: disable=import-error,no-name-in-module
+from utils.AnalysisUtils import ComputeCrossSection, GetPromptFDFractionCutSet
+from utils.StyleFormatter import SetGlobalStyle, SetObjectStyle, GetROOTColor
+
 systErr = AliHFSystErr()
 systErr.SetIsDataDrivenFDAnalysis(True)
 
+RYFractionCorrelation = 'corr'
+if sum([args.corr, args.uncorr, args.anticorr]) == 0:
+    print('WARNING: Raw Yield and feed-down uncertanties correlation is not specified.', 
+          'Options are --corr --uncorr --anticorr. Default is --corr.')
+elif sum([args.corr, args.uncorr, args.anticorr]) == 0:
+    print('ERROR: you cannot choose two correlation at the same time. Exit!')
+    sys.exit()
```

Comment:
The condition in the `if` will never happen (you have always `args.corr == 1`).
The condition in the `elif` is not doing what you write in the print.
Please rework this part